File: chatwoot/csharp/src/OSEG.ChatwootExamples/GetDetailsOfAInboxExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Com.Chatwoot.Api;
using Com.Chatwoot.Client;
using Com.Chatwoot.Model;

namespace OSEG.ChatwootExamples;

public class GetDetailsOfAInboxExample
{
    public static void Run()
    {
        var config = new Configuration();

        try
        {
            var response = new InboxAPIApi(config).GetDetailsOfAInbox(
                inboxIdentifier: null
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling InboxAPIApi#GetDetailsOfAInbox: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: chatwoot/typescript-axios/src/GetDetailsOfAInboxExample.ts
import axios, { AxiosError } from "axios";
import * as api from "chatwoot_client"

const configuration = new api.Configuration({
});

new api.InboxAPIApi(configuration).getDetailsOfAInbox(
  "inbox_identifier_string", // inboxIdentifier
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling InboxAPIApi#getDetailsOfAInbox:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
